bot/kodiak/queue.py

```diff
@@ -481,6 +483,7 @@ def __init__(self) -> None:
         self.worker_tasks: MutableMapping[
             str, tuple[Task[NoReturn], Literal["repo", "webhook"]]
         ] = {}  # type: ignore [assignment]
+        self.worker_task_heatbeats: MutableMapping[str, int] = {}
```

Comment:
need to figure out a way for the task to have access to this

Reply:
I wonder if there is an asyncio way to have the task thrown an exception unless it checks in every so often